asmcomp/cmmgen.ml

```diff
@@ -940,6 +941,11 @@ let rec expr_size env = function
       expr_size env closure
   | Usequence(_exp, exp') ->
       expr_size env exp'
+  | Uoffset (exp, offset) ->
+      (match expr_size env exp with
+      | RHS_block blocksize -> RHS_infix { blocksize; offset }
+      | RHS_nonrec -> RHS_nonrec
+      | _ -> assert false)
```

Comment:
You could also handle
```RHS_infix { blocksize; offset = offset' } -> RHS_infix { blocksize; offset = offset + offset' }```
I'm not certain that I can produce an example where Uoffsets are chained, but in theory, flambda could produce it through convoluted sequence of unrolling.